src/olympia/amo/monitors.py

```diff
@@ -224,3 +225,31 @@ def remotesettings():
         status = f'Failed to execute task in time: {e}'
         monitor_log.critical(status)
     return status, None
+
+
+def cinder():
+    # check cinder connectivity
+    signer_results = None
+    status = ''
+
+    cinder_api_url = settings.CINDER_SERVER_URL
+    if cinder_api_url:
+        try:
+            health_url = urljoin(cinder_api_url, '/health', allow_fragments=False)
+            response = requests.get(health_url, timeout=5)
+            if response.status_code != 200:
```

Comment:
Why do we care about 200 literally? Why not `raise_for_status()` and remove the if else? Just curious about the rationale.

Reply:
I was adopting a similar style to the other monitor functions but we can use `raise_for_status`